supercommand.go

```diff
@@ -514,6 +514,12 @@ func (c *SuperCommand) Run(ctx *Context) error {
 	}
 	err := c.action.command.Run(ctx)
 	if err != nil && !IsErrSilent(err) {
+		formatter := c.hasFormattingFlag()
```

Comment:
I'm unsure about this, I believe you could do this in `Init` and store it as a member.